modules/index/src/test/java/org/apache/ignite/internal/index/IndexBuildControllerTest.java

```diff
@@ -146,6 +156,7 @@ void testStartBuildIndexesOnIndexCreate() {
         createIndex(INDEX_NAME);
 
         verify(indexBuilder, never()).scheduleBuildIndex(
+                eq(zoneId(TABLE_NAME)),
```

Comment:
Will several zones be checked here? If not, then I suggest making the `zoneId()` method similar to `tableId()`.

Reply:
Thanks. Fixed